### PR TITLE
Version 0.20.2

### PR DESCRIPTION
**Pull request by aminalaee**
- https://github.com/encode/starlette/pull/1675
- https://github.com/encode/starlette/pull/1377

https://github.com/encode/starlette/releases/edit/untagged-04950ac69ee236fc5551

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 0.20.2
+
+May 31, 2022
```
Comment (Kludex) [inline docs/release-notes.md]: This will change - just as a reminder.

File: docs/release-notes.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+### Fixed
+* Allow colons when compiling Route and Mount [#1657](https://github.com/encode/starlette/pull/1657).
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
* Fix regression on route paths with colons [#1675](https://github.com/encode/starlette/pull/1675).
```

File: docs/release-notes.md (hunk 1 above)
Comment (Kludex) [reply docs/release-notes.md]:
```suggestion
June 4, 2022
```

**Review by Kludex — CHANGES_REQUESTED** [review]
Remember to share the GitHub release link. 🙏 

Thanks for this @aminalaee ! 🎉

**Comment by aminalaee** [comment]
@Kludex Done.

File: docs/release-notes.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
+## 0.20.2
+
+June 4, 2022
+
+### Fixed
+* Fix regression on route paths with colons [#1675](https://github.com/encode/starlette/pull/1675).
+* Allow staticfiles to follow symlinks [#1337](https://github.com/encode/starlette/pull/1377).
```
Comment (Kludex) [inline docs/release-notes.md]: Is this an addition or really a fix? I don't care. Just questioning in case.

**Review by Kludex — APPROVED** [review]

File: docs/release-notes.md (hunk 3 above)
Comment (aminalaee) [reply docs/release-notes.md]: Yeah it's a bit hard to tell, internally it was designed this way, but from the issue this is introduced as a bug. Open to suggestion though.

Comment (Kludex) [reply docs/release-notes.md]: Your call. Do as you wish.

File: docs/release-notes.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
+## 0.20.2
+
+June 4, 2022
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
June 7, 2022
```

File: docs/release-notes.md [hunk 5]
```diff
@@ -1,3 +1,11 @@
+## 0.20.2
+
+June 7, 2022
+
+### Fixed
+* Fix regression on route paths with colons [#1675](https://github.com/encode/starlette/pull/1675).
+* Allow staticfiles to follow symlinks [#1337](https://github.com/encode/starlette/pull/1377).
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
* Allow `StaticFiles` to follow symlinks [#1337](https://github.com/encode/starlette/pull/1377).
```

**Comment by Kludex** [comment]
Comparison against tag: https://github.com/encode/starlette/compare/0.20.1...master

**Pull request merged**